src/main/java/frc/robot/Constants.java

```diff
@@ -114,8 +115,9 @@ public static class constDrivetrain {
     public static CANcoderConfiguration CANCODER_CONFIG = new CANcoderConfiguration();
 
     static {
-      // TODO: im sure a lot of this can be cleaned up, but some of them are passed
-      // into the constructor of drive
+      // TODO: Im sure a lot of this can be cleaned up, but some of them are passed
```

Comment:
This comment probably didn't make initial sense to your because it wasn't a multiline comment;
Its supposed to be "... but some of them are passed into the constructor of Drivetrain"
Really nit-picky but I'd rather we just fix it now :p  maybe just make it a multiline comment